draft-ietf-scitt-architecture.md

```diff
@@ -327,23 +327,22 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
        v                 |  Identity   |
   .----+----.            |  Documents  |
  | Statement |            '---+--+----'
-  '----+----'       cose sign |  | cose verify
+  '----+----'                 |  |
        |    .----------------'|  |
        |   |                  |  |
        v   v                  |  |'----------.
   .----+---+---.              |  |            |
  |    Signed    |             |  |            |
  |   Statement  |             |  |            |
- | (COSE_Sign1) |             |  |            |
```

Comment:
This current document reflects shifting from using "envelope" to referencing COSE_Sign1. See #245 

Personally, I'd like to keep the current state for the diagram

Reply:
I agree with @SteveLasker : I would not overcomplicate the already cumbersome figure any more.

Lets drop this PR. We do not need it!